js/assessments/matchKeywordInSubheading.js

```diff
@@ -0,0 +1,51 @@
+var AssessmentResult = require( "../values/AssessmentResult.js" );
+
+/**
+ * Returns a score and text based on the keyword matches object.
+ *
+ * @param {object} subheadings The object with all subheadings matches.
+ * @param {object} i18n The object used for translations
+ * @returns {object} resultObject with score and text
+ */
+var calculateKeywordMatchesResult = function( subheadings, i18n ) {
```

Comment:
I would write `subheadings` as `subHeadings` to be consistent with `resultObject`